proto/http/http.proto

```diff
@@ -149,6 +147,12 @@ message HTTPRequest {
   Request request = 2;
 }
 
+message HttpResponse {
+  // header is sent to synchronize any header modifications from the plugin
+  // process
+  repeated Element header = 1;
```

Comment:
grpc requires a header to indicate the end of an rpc, which is directly set onto the map in the plugin process ([ref](https://github.com/grpc/grpc-go/blob/cd05c9e58fb9d6563e1c7d895f55faec763fc20e/internal/transport/handler_server.go#L244)).